apps/site/pages/docs/global-tokens/typography.mdx

```diff
@@ -160,9 +162,14 @@ export const mobileList = [
 
 ---
 
-## Add a custom font
+## Adding a custom font
+
+Using a custom font helps to keep a cohesive brand across your store. 
+To maintain readability in your store, choose an easily readable font and the appropriate color and text size.
+Follow the steps below to add a custom font to your store.
 
-1. Create a new `WebFonts.tsx` file in `src/fonts` folder, and add a `<link>` tag for your `font-family` of choice.
+1. Go to the `src` folder and create the `fonts` folder.
```

Comment:
Could you use the `Steps` component here?

Reply:
hmmm, I don't think we are using `Steps` components in our doc for this kind of scenario 🤔 
We are currently using for Before you start and Requirement sections.
@Mariana-Caetano WDYT?

Reply:
I suggest we keep the steps as they are in this guide. We can use the `Steps` component in either the `Before you Start` or `Requirements` section, as Fanny pointed out. Make it a pattern throughout the documentation. Since these sections (`Before you Start` and `Requirements`) are essential to begin the tutorials/guides, this component effectively highlights their importance.

Reply:
@renatamottam